ynput/OpenPype
# Max: Extract Max Scene Raw should include OP metadata

## Changelog Description
Extract Max Scene Raw would publish the container with OP metadata. Once the user load the published objects, the mesh objects won't be parented to the container while it would store the OP data which links to the published objects.
## Additional info
Affecting #5396 and slightly affecting #5378
<del>Should be merged after #5378 </del>
There is a room for discussion on whether the old duplicates should be deleted. If the "delete old duplicates" implemented for  switching or updating version, the old container would only includes the objects which is not duplicated in the newer version.
Please create new instance to publish something for test the loader.
## Testing notes:
1. Publish something with MaxScene Instance with more than 1 version
2. Load the instance
3. The container should include OP parameter
4. Switch the version or update the version
5. It should load the duplicated objects with the new container which also stores the data
